docs/monitoring.md

```diff
@@ -176,6 +176,7 @@ Each instance can report to zero or more _sinks_. Sinks are contained in the
 * `JmxSink`: Registers metrics for viewing in a JMX console.
 * `MetricsServlet`: Adds a servlet within the existing Spark UI to serve metrics data as JSON data.
 * `GraphiteSink`: Sends metrics to a Graphite node.
+* `Slf4jSink`: Sends metrics to log4j as logging entries
```

Comment:
Intended to say log4j because that's the final destination. (the pipeline is metrics data -> slf4j -> log4j), but can see why it's confusing. Updating string to say "send metrics to log4j via slf4j" to be clear.